CHANGELOG.md

```diff
@@ -27,15 +35,29 @@ Change Log
       - `Initializer.Config.newWallet`
       - `Initializer.Config.setViewingKeys`
 - `cash.z.ecc.android.sdk`:
-  - `Synchronizer.shieldFunds` now takes a transparent account private key (representing
-    all transparent secret keys within an account) instead of a transparent secret key.
+  - `Synchronizer.Companion.new` now takes an optional `seed` argument. This should be
+    provided if `Synchronizer.Companion.new` throws an error that a database migration
+    requires the wallet seed.
```

Comment:
This was accurate as of the commit where I added the `seed` argument, but now that `Initializer` has been removed, it needs to be rewritten.